scripts/http-form-brute.nse

```diff
@@ -455,7 +458,11 @@ Driver = {
 
   login = function (self, username, password)
     local response, success = self:submit_form(username, password)
-    if not response then
+    if response and not response.status then
```

Comment:
I've added check here for the socket error in _response.status_ but it's not only _socket.error_  which may cause old to be nil. There are few more calls to update_cookies from the various part of script.